### ISSUE TITLE
loklak.fossasia.org not working/loading

### ISSUE THREAD

**Issue by jig08** [fossasia/loklak_search#90]
Console log.

```

Uncaught Error: Template parse errors:
Unexpected closing tag "a" ("XPwD/AVER/qFOTZZGAAAAAElFTkSuQmCC" alt="loklak social media search engine">
         </a>">
        [ERROR ->]</a>
      </div>
      <div id="navbar" class="navbar-collapse collapse">
"): ContactComponent@12:8


```

<img width="1280" alt="screen shot 2016-11-02 at 12 20 59 pm" src="https://cloud.githubusercontent.com/assets/9291400/19919153/d8795088-a0f6-11e6-81b5-8b94abe8f634.png">

**Comment by hemantjadon** [comment]
Fixed just now in #89

**Comment by jig08** [comment]
@hemantjadon thanks.
